### ISSUE TITLE
This repo is outdated and abandoned 

### ISSUE THREAD

**Issue by Mati365** [ActiveCampaign/activecampaign-api-nodejs#68]
Current version of ActiveCampaign API is NOT supported by this package and there is no info about this in docs. Do not use it in your project!

**Comment by martinsotirov** [comment]
Thanks for the heads up. We were about to start integrating Active Campaign in our project and were wondering if we can use this vs their REST API directly.

**Comment by theDanielJLewis** [comment]
Is there any good alternative? It seems all the AC API packages on npmjs are old.

**Comment by Mati365** [comment]
@theDanielJLewis nope. In my current company, we managed to write our own tiny client with a subset of the endpoints we really wanted to support.

**Comment by brucruz** [comment]
Thank you all for sharing this!
@Mati365, did you and your current company consider to open source this tiny client you mentioned? I would be more than glad to help maintain it.
